examples/tutorial/02_Plotting.ipynb

```diff
@@ -563,7 +563,105 @@
    "source": [
     "Here it is clear that the most commonly detected events are light, and typically shallow.\n",
     "\n",
-    "# Exploring further\n",
+    "## Output Matplotlib or Plotly plots\n",
+    "\n",
+    "While the default plotting backend/extension of hvPlot is Bokeh, it also supports rendering plots with either [Plotly](https://plotly.com/python/) or [Matplotlib](https://matplotlib.org/). As Bokeh, Plotly is an interactive library that provides similar data exploring actions (pan, hover, zoom, etc.). You can decide to use Plotly instead of Bokeh if you prefer its look and feel, or if your organization's policy is to use that plotting library. Matplotlib is mostly a static plotting library, it is great at creating static images (e.g. PNG) that you can then easily save and share with your friends or coworkers, and embed in documents for publication.\n",
```

Comment:
```suggestion
    "While the default plotting backend/extension of hvPlot is Bokeh, it also supports rendering plots with either [Plotly](https://plotly.com/python/) or [Matplotlib](https://matplotlib.org/). Like Bokeh, Plotly is an interactive library that provides similar data-exploring actions (pan, hover, zoom, etc.). You can decide to use Plotly instead of Bokeh if you prefer its look and feel, or if your organization's policy is to use that plotting library. Matplotlib is supported for static plots only (e.g. PNG or SVG), which is useful for saving and sharing images and embedding them in documents for publication.\n",
```